lib/lndclient/LndClient.ts

```diff
@@ -597,6 +596,38 @@ class LndClient extends SwapClient {
     }
   }
 
+  public lookupPayment = async (rHash: string) => {
```

Comment:
What do you think about adding an integration test to this once since this is a new public method that is full of logic?

Reply:
I felt there wasn't much for me to test in the LndClient case (we always just make a single grpc call and return a value based on that call), but I did add cases for the Raiden `lookupPayment` logic since it is a bit more involved.